api/test/h3-data/h3-data.spec.ts

```diff
@@ -104,7 +104,7 @@ describe('H3-Data Module (e2e)', () => {
       );
     });
 
-    test('When I query H3 data at minimal resolution, then I should get 8 h3indexes', async () => {
+    test.only('When I query H3 data at minimal resolution, then I should get 8 h3indexes', async () => {
```

Comment:
why the "only()" ?

Reply:
Leftover of refactoring test, good catch, thanks!